Dockerfile

```
@@ -35,6 +35,7 @@
     curl \
     nodejs \
     npm \
+    libcap2-bin \  # provides setcap
     && rm -rf /var/lib/apt/lists/* \
```

Hadolint (2.12.0)

[error] unexpected '&'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)